Source/Bugsnag.h

```diff
@@ -183,6 +183,33 @@ static NSString *_Nonnull const BugsnagSeverityInfo = @"info";
  */
 + (void)leaveBreadcrumbForNotificationName:(NSString *_Nonnull)notificationName;
 
+/**
+ * Leave a "breadcrumb" log message, representing an action that occurred
+ * in your app.
+ *
+ * @param message The log message to leave
+ * @param metadata Additional metadata included with the breadcrumb
+ *
+ * , to aid with debugging, along with additional metadata.
+ */
++ (void)leaveBreadcrumbWithMessage:(NSString *_Nonnull)message
```

Comment:
The notifier spec states that `metadata` and `type` are optional parameters. I would interpret that by implementing the following two methods:

`leaveBreadcrumb:message`
`leaveBreadcrumb:message:metadata:type`

So I think it'd be ok to remove this particular method variant, unless there are strong reasons for including it.

Reply:
As discussed this will be removed.